diff --git a/install/config.php b/install/config.php
index fe9f8fe..57fe015 100644
--- a/install/config.php
+++ b/install/config.php
@@ -18,31 +18,32 @@ return array(
     'addons' => array(),

     'cart_settings' => array (
-        'email' => 'admin@example.com',
-        'password' => 'admin',
-        'secret_key' => 'YOURVERYSECRETCEY',
+        'email' => 'YOUR@ADMIN.EMAIL',
+        'password' => 'YOUR-ADMIN-PASSWORD',
+        'secret_key' => 'YOUR-RANDOM-HASH',
         'languages' => array (
-            'en', 'da', 'de', 'es', 'fr', 'el', 'it', 'nl', 'ro', 'ru', 'bg', 'no', 'sl',
+            'en', 'de',
         ),
         'main_language' => 'en',
         'demo_catalog' => true,
         'theme_name' => 'bright_theme',
-        'license_number' => 'CART-1111-1111-1111-1111'
+        'license_number' => 'CART-1111-1111-1111-1111',
+	    'feedback_auto' => 'N'
     ),
     'database_settings' => array(
-        'host' => 'localhost',
-        'name' => '%DB_NAME%',
-        'user' => '%DB_USER%',
-        'password' => '%DB_PASS%',
+        'host' => 'DATABASE_HOST',
+        'name' => 'DATABASE_NAME',
+        'user' => 'DATABASE_USER',
+        'password' => 'DATABASE_PASSWORD',
         'table_prefix' => 'cscart_',
         'database_backend' => 'mysqli',
         'notify' => false,
         'allow_override' => 'Y',
     ),
     'server_settings' => array (
-        'http_host' => '%HTTP_HOST%',
+        'http_host' => 'YOUR_DOMAIN',
         'http_path' => '',
-        'https_host' => '%HTTP_HOST%',
+        'https_host' => 'YOUR_DOMAIN',
         'https_path' => '',
         'correct_permissions' => true,
     ),
